cs251tk/toolkit/args.py

```diff
@@ -157,6 +158,8 @@ def process_args():
     parser = build_argparser()
     args = vars(parser.parse_args())
 
+    logging.basicConfig(level=logging.DEBUG if debug else logging.WARNING)
```

Comment:
Isn't this always going to be true, since `debug` is a function at this point?
```suggestion
    logging.basicConfig(level=logging.DEBUG if args['debug'] else logging.WARNING)
```

Reply:
Forgot to test it without the debug flag. You're correct.